src/parser/classes/DecoratedAvatarView.ts

```diff
@@ -0,0 +1,86 @@
+import { YTNode } from '../helpers.js';
+import type { RawNode } from '../index.js';
+import { NavigationEndpoint } from '../nodes.js';
+
+export default class DecoratedAvatarView extends YTNode {
+  static type = 'DecoratedAvatarView';
+
+  avatar: {
+      avatar_view_model: {
```

Comment:
@absidue do you mean that avatar_view_model needs to be of type AvatarView?

Reply:
Yes. Ideally, there should be an AvatarView node. Anything item that has "model" or "renderer" in its name should have its own individual module. The generator can only recognize renderers it seems.